### ISSUE TITLE
Provide a setting for room name

### ISSUE THREAD

**Issue by jo-so** [tijder/SmsMatrix#54]
I would like to have *(SMS)* in the room name to distinguish the rooms from the other rooms for Whatsapp or Matrix. If you could add the text field for a room name on initialization. It might be empty or with placeholders `%n (SMS)` where *%n* gets replace by the name.

**Comment by xpseudonym** [comment]
Yes, if the contact name can be provided in the message window and the contact number provided in the room>settings>topic field then surely these data can be used to create a more meaningful room name than 'Empty Room'? Thanks

**Comment by xpseudonym** [comment]
Issue #38 seems connected to this, ie identify rooms by contact information: use contact pic, use contact name - unless other party isn't in contacts in which case use other party's phone number to name and identify room
